c/parallel/include/cccl/c/types.h

```diff
@@ -87,4 +87,10 @@ typedef struct cccl_iterator_t
   void* state;
 } cccl_iterator_t;
 
+typedef enum cccl_sort_order_t
+{
+  CCCL_ASCENDING  = 0,
+  CCCL_DESCENDING = 1
```

Comment:
If a coma is added after `CCCL_ITERATOR = 1` per style guidelines, the trailing coma is needed here too